stdlib/experimental/influxdb/influxdb.flux

```diff
@@ -0,0 +1,18 @@
+package influxdb
+
+// api submits an HTTP request to the specified API path.
+// Returns HTTP status code, response headers, and body as a byte array.
+builtin api : (
+	method: string,
+	path: string,
+	?host: string,
+	?token: string,
+	?body: string,
+	?headers: A,
+	?query: B,
+	?timeout: duration
+) => {
+	statusCode: int,
+	body: bytes,
+	headers: C
+} where A: Record, B: Record, C: Record
```

Comment:
I think we should use a `dict` type instead of a record type here as headers and query must be strings.
Also the body should be bytes on input as well as output.

So the type would be 

```
builtin api : (
	method: string,
	path: string,
	?host: string,
	?token: string,
	?body: bytes,
	?headers: [string: string],
	?query: [string: string],
	?timeout: duration
) => {
	statusCode: int,
	body: bytes,
	headers: [string: string],
} 
```

Reply:
I based this implementation off of the `experimental/http.get` implementation, which used a record. I think that implementation might have predated the dictionary type. I can investigate this idea.

Reply:
Implemented this change, thanks again!